### ISSUE TITLE
Workspace thinks arrays are unsupported types

### ISSUE THREAD

**Issue by spyder-issues-migrator** [spyder-ide/spyder#175]
_From [anatoliy...@gmail.com](https://code.google.com/u/101136359641652000647/) on 2010-03-12T17:16:57Z_

What steps will reproduce the problem?

 1. Type a = np.array(1) in the Interactive Console 2. 3. What is the expected output? What do you see instead? It should display the array 'a' in the interactive console, but it does
not.  I can see if if I disable "Exclude unsupported data types" What version of the product are you using? On what operating system? This persists on 1.0.3 and 1.1alpha2, using Python2.6.  My OS is WinXP on a
VMplayer inside Ubuntu 8.04.  I tried this on a native Windows installation
and the workspace is perfectly fine there.  I tried installing all the
packages I thought might be relevant(IPython, PyLint, wxWidgets), but
nothing changed.  I really don't know what's going on here, it's hard to
believe the VM is at fault.  Please let me know if I can help you narrow it
down in any way. 

Please provide any additional information below

.

_Original issue: http://code.google.com/p/spyderlib/issues/detail?id=175_

**Comment by spyder-issues-migrator** [comment]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2010-03-30T04:38:37Z_

The workspace is using a supported types list which is configured only once, at the 
first launch of Spyder -- i.e. if at the first launch you hadn't numpy installed, it 
will never show numpy arrays in workspace even if you install numpy afterwards. To 
solve this situation, you simply have to reset the Spyder configuration file by doing 
the following:
1. launch Spyder with special option --reset:
spyder --reset
2. quit Spyder and relaunch it again with your own usual options.

**Status:** Accepted

**Comment by spyder-issues-migrator** [comment]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2010-06-19T05:39:24Z_

**Status:** WontFix
